#! /bin/sh /usr/share/dpatch/dpatch-run
## opt-92_dvbplayer-no-busy.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: http://www.vdr-portal.de/board/thread.php?postid=962639#post962639

@DPATCH@
diff -urNad vdr-1.7.16~/dvbplayer.c vdr-1.7.16/dvbplayer.c
--- vdr-1.7.16~/dvbplayer.c	2010-03-07 15:24:26.000000000 +0100
+++ vdr-1.7.16/dvbplayer.c	2010-12-22 22:01:50.447380867 +0100
@@ -407,9 +407,9 @@
            cPoller Poller;
            DevicePoll(Poller, 10);
            Sleep = false;
+           }
            if (playMode == pmStill || playMode==pmPause)
               cCondWait::SleepMs(3);
-           }
         {
           LOCK_THREAD;
